### PR TITLE
va-icon: Adding new icons 6/6/24

### PR DESCRIPTION
**Pull request by Andrew565**
## Chromatic

https://2872-adding-icons-6-6-24--65a6e2ed2314f7b8f98609d8.chromatic.com

---

## Description
Closes [#2872](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2872)

## QA Checklist
- [X] Component maintains 1:1 parity with design mocks
- [X] Text is consistent with what's been provided in the mocks
- [X] Component behaves as expected across breakpoints
- [X] Accessibility expert has signed off on code changes (if applicable. If not applicable provide reason why)
- [X] Designer has signed off on changes (if applicable. If not applicable provide reason why)
- [X] Tab order and focus state work as expected
- [X] Changes have been tested against screen readers (if applicable. If not applicable provide reason why)
- [X] New components are covered by e2e tests; updates to existing components are covered by existing test suite
- [X] Changes have been tested in vets-website using Verdaccio (if applicable. If not applicable provide reason why)

## Screenshots
N/A

## Acceptance criteria
- [X] QA checklist has been completed
- [X] Screenshots have been attached that cover desktop and mobile screens

## Definition of done
- [X] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Review by powellkerry — APPROVED** [review]

File: packages/storybook/public/img/sprite.svg [hunk 1]
```diff
@@ -1 +1,3 @@
...
+  <path d="M20 8H17V4H3C1.9 4 1 4.9 1 6V17H3C3 18.66 4.34 20 6 20C7.66 20 9 18.66 9 17H15C15 18.66 16.34 20 18 20C19.66 20 21 18.66 21 17H23V12L20 8ZM6 18.5C5.17 18.5 4.5 17.83 4.5 17C4.5 16.17 5.17 15.5 6 15.5C6.83 15.5 7.5 16.17 7.5 17C7.5 17.83 6.83 18.5 6 18.5ZM19.5 9.5L21.46 12H17V9.5H19.5ZM18 18.5C17.17 18.5 16.5 17.83 16.5 17C16.5 16.17 17.17 15.5 18 15.5C18.83 15.5 19.5 16.17 19.5 17C19.5 17.83 18.83 18.5 18 18.5Z"/>
```
Comment (danbrady) [inline packages/storybook/public/img/sprite.svg:2]: @Andrew565 Not a huge deal, but why are there a few line breaks in this minified file?

File: packages/storybook/public/img/sprite.svg [hunk 2]
```diff
@@ -1 +1,3 @@
...
...d="m9 17 3-2.94a9.34 9.34 0 0 0-1-.06c-2.67 0-8 1.34-8 4v2h9l-3-3zm2-5c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4m4.47 8.5L12 17l1.4-1.41 2.07 2.08 5.13-5.17 1.4 1.41z"/></symbol><symbol id="inbox" viewBox="0 -960 960 960"><path d="M200-120q-33 0-56.5-23.5T120-200v-560q0-33 23.5-56.5T200-840h560q33 0 56.5 23.5T840-760v560q0 33-23.5 56.5T760-120H200Zm280-200q38 0 69-22t43-58h168v-360H200v360h168q12 36 43 58t69 22Z"/></symbol><symbol id="key" viewBox="0 -960 960 960"><path d="M280-360q50 0 85-35t35-85q0-50-35-85t-85-35q-50 0-85 35t-35 85q0 50 35 85t85 35Zm0 120q-100 0-170-70T40-480q0-100 70-170t170-70q81 0 141.5 46T506-560h335l79 79-140 160-100-79-80 80-80-80h-14q-25 72-87 116t-139 44Z"/></symbol><symbol id="local" viewBox="0 -960 960 960"><path d="M240-160q-50 0-85-35t-35-85H40v-440q0-33 23.5-56.5T120-800h560v160h120l120 160v200h-80q0 50-35 85t-85 35q-50 0-85-35t-35-85H360q0 50-35 85t-85 35Zm0-80q17 0 28.5-11.5T280-280q0-17-11.5-28.5T240-320q-17 0-28.5 11.5T200-280q0 17 11.5 28.5T240-240Zm480 0q17 0 28.5-11.5T760-280q0-17-11.5-28.5T720-320q-17 0-28.5 11.5T680-280q0 17 11.5 28.5T720-240Zm-40-200h170l-90-120h-80v120Z"/></symbol><symbol id="local_shipping" viewBox="0 0 24 24">
```
Comment (danbrady) [inline packages/storybook/public/img/sprite.svg:1]: @Andrew565 This has a "local" icon which doesn't exist. Should be "local_shipping" (which is already there).

File: packages/storybook/public/img/sprite.svg (hunk 1 above)
Comment (Andrew565) [reply packages/storybook/public/img/sprite.svg:2]: Not sure why, to be honest. If I had to guess I'd guess that there's an artificial line length limit that some programmer thought was valuable to adhere to.

File: packages/storybook/public/img/sprite.svg (hunk 2 above)
Comment (Andrew565) [reply packages/storybook/public/img/sprite.svg:1]: I removed the "local" icon from the Storybook story, as well as the sprite repo (dst-uswds-compile), but this PR had already gone out by then, so I chose to just leave the sprite in the sprite sheet for now. Doubtful anyone will notice it, and it'll get removed the next time the sprite sheet is updated.

**Pull request merged**